pkgdb/Makefile

```diff
@@ -284,6 +283,15 @@ toml_CFLAGS ?=                                                                \
   $(shell $(NIX) build --no-link --print-out-paths 'nixpkgs#toml11')/include
 toml_CFLAGS := $(toml_CFLAGS)
 
+ifeq (Linux,$(OS))
+sentry_PREFIX ?=                                                        \
+	$(shell $(NIX) build --no-link --print-out-paths '.#sentry-native')
+sentry_PREFIX := $(sentry_PREFIX)
```

Comment:
nonblocking: what does this line do?

Reply:
Woops... apparently nothing.  Likely some debug or test code.